src/store/pageview.ts

```diff
@@ -17,6 +26,29 @@ class ViewStore {
     storage.set(StorageKeys.PAGE_VIEWS, this.pageViews);
   }
 
+  _trim() {
+    this.pageViews.sort((a: PageView, b: PageView): number => b.ts - a.ts);
+    this.pageViews = this.pageViews.filter(
+      (pageView: PageView, i: number) =>
+        pageView.ts > timeStampInSecs() - this.maxAge && i < this.storageSize
```

Comment:
This will make a new timestamp on each iteration, we can make it once and use that over and over.

Reply:
You're right!
I'll change it.